site.yml

```diff
@@ -104,15 +104,15 @@ asciidoc:
     std-port-mysql: '3306'
     std-port-redis: '6379'
 #   ocis
-    latest-ocis-version: '4.0'
-    previous-ocis-version: 'next'
+    latest-ocis-version: '5.0'
+    previous-ocis-version: '4.0'
     # these versions are just for printing like in releases but not used for referencing
     # needs to be changed here and not in the docs-ocis repo to be properly shown on the web
     # the following versions get printed on <all> build versions where applicapable
     # for branch only dependent version content, change the values in antora.yml at compose_tab_1_tab_text
-    ocis-actual-version: '4.0.5'
-    ocis-former-version: '3.0.0'
-    ocis-compiled: '2023-10-06 00:00:00 +0000 UTC'
+    ocis-actual-version: '5.0.0'
+    ocis-former-version: '4.0.5'
+    ocis-compiled: '2024-02-23 00:00:00 +0000 UTC'
```

Comment:
Some future "prophecy"

Reply:
yes, the compiled thing can be changes at any time later to the correct date, the versions must be correct now.